src/app/lib/supabase/server.ts

```
@@ -2,28 +2,28 @@ import { createServerClient } from "@supabase/ssr";
-            );
-          } catch {
-            // The `setAll` method was called from a Server Component.
-            // This can be ignored if you have middleware refreshing
-            // user sessions.
-          }
-        },
-      },
-    }
-  );
+	return createServerClient(
+		process.env.NEXT_PUBLIC_SUPABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)